tests/unit_tests/tests_launch/test_launch_cli.py

```diff
@@ -510,6 +510,23 @@ def print_then_exit():
         assert "except caught, acked item" in result.output
 
 
+# def test_launch_bad_api_key(
```

Comment:
Is this commented because our tests infra changed?

Reply:
This is commented out because I am not sure how to effectively test the correctness of an API key in the testing suite. Despite passing in a broken key the error is never raised (in the test).